test/tools/libtesteth/BlockChainHelper.h

```diff
@@ -137,10 +137,16 @@ class TestBlock
 class TestBlockChain
 {
 public:
-    TestBlockChain() : TestBlockChain(defaultGenesisBlock()) {}
-    TestBlockChain(TestBlock const& _genesisBlock);
-
-    void reset(TestBlock const& _genesisBlock);
+    enum class MiningType
+    {
+        Ethash,
+        NoProof,
+        Default
```

Comment:
I'd add a comment that `Default` means "don't override the engine from given config"

Reply:
fixed